downstream/modules/platform/proc-gs-eda-set-up-rulebook-activation.adoc

```diff
@@ -0,0 +1,57 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-09-24
+:_mod-docs-content-type: PROCEDURE
+
+[id="gs-eda-set-up-rulebook-activation_{context}"]
+= Setting up a rulebook activation
+
+.Prerequisites
+
+* You have set up a project.
+* You have set up a decision environment.
+* You have set up an {ControllerName} token. See link:{URLEDAUserGuide}/index#eda-set-up-token-to-authenticate[Setting up an {ControllerName} token] in the {TitleEDAUserGuide} guide for more information.
+
+.Procedure
+. From the navigation panel, select {MenuADRulebookActivations}.
+. Click btn:[Create rulebook activation]. 
+. Enter the following information:
+* *Name*: Insert the name.
+* *Description*: This field is optional.
+* *Organization*: This field is optional.
+* *Project*:  This field is optional.
+* *Rulebook*: Rulebooks are displayed according to the project you selected.
+* *Credential*: Select 0 or more credentials for this rulebook activation. This field is optional.
++
+NOTE: The credentials that display in this field are customized based on your rulebook activation and only include the following credential types: Vault, {PlatformName}, or any custom credential types that you have created. For more information about credentials, see link:{URLControllerUserGuide}/index#eda-set-up-credential[Setting up credentials for {EDAcontroller}] in the {TitleControllerUserGuide} guide.
```

Comment:
Notes should really be formatted as follows to ensure there are no issues when it builds:
[NOTE]
====
<note text>
====